### PR TITLE
Fixes a csadmin test

### PR DESCRIPTION
**Pull request by nkcr**
Commit 79ddf19d2ccb41ffa97a7595e923b5b4aab068a2 included a test that was only compatible with a feature implemented in the odyssey-needs branch.

This PR makes the test pass by adding this missing feature.

---

🙅‍ Friendly checklist:

- [x] 0. Code comments are added (or updated) when/where needed and explain the WHY of the code.
- [x] 1. Design choices, user documentation and any additional doc are added (or updated) in READMEs.
- [x] 2. Any new behaviour is tested and small units of code that can be are unit tested.
- [x] 3. Code comments are added on tests to explain what they do.
- [x] 4. Errors are systematically wrapped with a meaningful message using `xerrors.Errorf` and the `%v` verb.
- [x] 5. Hard limit of 80 chars is always respected.
- [x] 6. Changes are backward compatible.
- [x] 7. Indentation level does not exceed 5, although 4 is already suspicious.
- [x] 8. Functions, files, and packages are kept to a manageable size and decomposed into smaller units if needed.
- [x] 9. There are no magic values.

### CODE REVIEW

**Comment by nkcr** [comment]
thanks @gnarula for pointing that out.

File: calypso/csadmin/clicontracts/write.go [hunk 1]
```diff
@@ -230,6 +230,19 @@ func WriteGet(c *cli.Context) error {
 		return errors.New("didn't get a write instance: " + err.Error())
 	}
 
+	if c.Bool("export") {
+		_, buf, _, _, err := proof.KeyValue()
+		if err != nil {
+			return errors.New("failed to get value from proof: " + err.Error())
```
Comment (gnarula) [inline calypso/csadmin/clicontracts/write.go]: Can we use `xerrors.Errorf("...: %v", err)` here please to be consistent with #2116

File: calypso/csadmin/clicontracts/write.go [hunk 2]
```diff
@@ -230,6 +230,19 @@ func WriteGet(c *cli.Context) error {
...
+		}
+		reader := bytes.NewReader(buf)
+		_, err = io.Copy(os.Stdout, reader)
+		if err != nil {
+			return errors.New("failed to copy to stdout: " + err.Error())
```
Comment (gnarula) [inline calypso/csadmin/clicontracts/write.go]: Similar to above, but perhaps a `return cothority.ErrorOrNil(err, "failed to copy to stdout")`

Comment (nkcr) [reply calypso/csadmin/clicontracts/write.go]: I am still not familiar with those custom things. Why should I use `cothority.ErrorOrNil` in this case?

**Pull request merged**